src/components/EmailNotifications/ExistingEmailSubscription.js

```diff
@@ -0,0 +1,131 @@
+import React, { useCallback } from 'react'
+import styled from 'styled-components'
+import { Button, GU, Link, textStyle, useTheme } from '@aragon/ui'
+import LocalIdentityBadge from '../LocalIdentityBadge/LocalIdentityBadge'
+import { addressesEqual, transformAddresses } from '../../lib/web3-utils'
+
+import emailNotifcationIllustration from '../../../src/assets/emailNotifications.svg'
+
+function ExistingEmailSubscription({
+  account,
+  compactMode,
+  onOptOut,
+  onSubscribeToNotifications,
+}) {
+  const theme = useTheme()
+
+  const textContent = `We’ve detected an email associated to the account ${account}.
+  First, we need to verify that you are in control of this account and then, authenticate the email address you provided.`
+
+  const handleOnSubscribeToNotifications = useCallback(() => {
+    onSubscribeToNotifications()
+  }, [onSubscribeToNotifications])
+
+  const handleOnOptOut = useCallback(() => {
+    onOptOut()
+  }, [onOptOut])
+
+  return (
+    <>
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+          padding-top: ${3 * GU}px;
+        `}
+      >
+        <div
+          css={`
+            display: flex;
+            flex-direction: column;
+            text-align: center;
+          `}
+        >
+          <div
+            css={`
+              align-items: center;
+            `}
+          >
+            <img
+              src={emailNotifcationIllustration}
+              width={141}
+              height={141}
+              alt=""
+            />
+          </div>
+          <span
+            css={`
+              ${textStyle('title2')};
+              margin-top: ${4 * GU}px;
+            `}
+          >
+            Stay up to date with email notifications
+          </span>
+
+          <span
+            css={`
+              ${textStyle('body2')};
+              color: ${theme.surfaceContentSecondary};
+              margin-top: ${1.5 * GU}px;
+            `}
+          >
+            {transformAddresses(textContent, (part, isAddress, index) =>
+              isAddress ? (
+                <span title={part} key={index}>
+                  <LocalIdentityBadge
+                    connectedAccount={addressesEqual(part, account)}
+                    entity={part}
+                    compact
+                  />
+                </span>
+              ) : (
+                <span key={index}>{part}</span>
+              )
+            )}
+          </span>
+        </div>
+        <span
+          css={`
+            ${textStyle('body2')};
+            color: #9096b6;
+            text-align: center;
+            margin-top: ${3 * GU}px;
+          `}
+        >
+          You have previously agreed to Aragon court{' '}
+          <Link href="#">legal terms </Link> and{' '}
+          <Link href="#"> email collection policy.</Link>
```

Comment:
```suggestion
          <Link href="#">email collection policy</Link>.
```

Also see a previous PR for the links.